sklearn/cluster/hierarchical.py

```diff
@@ -196,7 +197,8 @@ def ward_tree(X, connectivity=None, n_clusters=None, return_distance=False):
     else:
         if n_clusters > n_samples:
             raise ValueError('Cannot provide more clusters than samples. '
-                             '%i n_clusters was asked, and there are %i samples.'
+                             '%i n_clusters was asked, and there are'
```

Comment:
just to be sure. was it not PEP8 previosuly. If it was less than 80 characters avoid to make a diff for nothing.